test/SlowTests/Server/Replication/ReplicationCleanTombstones.cs

```diff
@@ -879,5 +862,53 @@ public async Task CanBackupAndRestoreTombstonesWithSameId(Options options)
                 }
             }
         }
+
+        private async Task<long> CreateAndRunBackup(IDocumentStore store, RavenDatabaseMode mode, string backupPath)
```

Comment:
I think we should move this to the infra as well

Reply:
`CreateAndRunBackupAsync`

Reply:
done